doc/whats_new/v1.0.rst

```diff
@@ -823,6 +823,14 @@ Changelog
   :func:`~sklearn.utils.check_array` in 1.0 and will raise a `TypeError` in
   1.2. :pr:`20165` by `Thomas Fan`_.
 
+:mod:`sklearn.gaussian_process`
+.........................
```

Comment:
```suggestion
:mod:`sklearn.gaussian_process`
...............................
```

Reply:
We will let this entry here for the moment but we will have to move it in a section 1.0.1 most probably.